__testUtils/mockPersonResource.js

```diff
@@ -29,14 +34,28 @@ export const aliceName = "Alice";
 export const aliceNick = "A";
 export const alicePhoto = "http://example.com/alice.jpg";
 
+const VCARD_WEBID_PREDICATE = "https://www.w3.org/2006/vcard/ns#WebId";
+
+export function mockWebIdNode(webId) {
+  const dataset = chain(
```

Comment:
I would not call this `dataset`, since that can be confused with the type `SolidDataset`. Maybe `webIdNode` or `webIdThinng` would be better?